trinity/common/config.py

```diff
@@ -83,7 +83,6 @@ class ModelConfig:
     max_response_tokens: int = 2048
     # The checkpoint directory, contains a latest dir link and multiple checkpoint dirs.
     checkpoint_path: str = ""
-    load_checkpoint: bool = True
```

Comment:
Check the example configs in `scripts/config`, most of them have this field

Reply:
OK, removed